src/index.js

```diff
@@ -0,0 +1,42 @@
+import './style.css';
+import id from './modules/uniqueId';
+import UI from './modules/render';
+
+const postUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games';
+const submitForm = document.querySelector('.btn-submit');
+const refreshBtn = document.querySelector('.btn-ref');
+const user = document.getElementById('name');
+const score = document.getElementById('number');
+
+UI.getData();
+// Post to the API
+const pastGameScores = async () => {
```

Comment:
- [ ] [optional] It would have been nice if the function name was `postGameScore`(I think you misspelled it) or a more descriptive name.